src/recon_buildblock/LogcoshPrior.cxx

```diff
@@ -506,16 +498,14 @@ LogcoshPrior<elemT>::parabolic_surrogate_curvature(DiscretisedDensity<3,elemT>&
             for (int dx=min_dx;dx<=max_dx;++dx)
             {
               // psi'(t)/t = tanh/t
-              elemT temp = fabs(current_image_estimate[z][y][x] - current_image_estimate[z+dz][y+dy][x+dx]);
-              elemT current  = weights[dz][dy][dx] * (1/(this->scalar))* surrogate(this->scalar*temp);
+              elemT temp =current_image_estimate[z][y][x] - current_image_estimate[z+dz][y+dy][x+dx];
+              elemT current = weights[dz][dy][dx] * (1/(this->scalar))* surrogate(this->scalar * temp);
 
```

Comment:
i'm not sure if there should be an additional 1/scalar in front here.  surrogate is already tanh(x)/x, so what this outputs is  weight/scalar * tanh(x * scalar) / (x * scalar) =   weight * tanh(x * scalar) / x. is that correct?

Reply:
Yep, this is wrong, there should be not 1/scalar. It is also wrong for the taylor case too.